lib/xboxkrnl/xboxkrnl.h

```diff
@@ -1801,6 +1823,16 @@ XBAPI VOID NTAPI RtlZeroMemory
     IN SIZE_T Length
 );
 
+#else
+
+#include <string.h>
+
+#define RtlFillMemory(Destination, Length, Fill) ((void)memset((Destination), (Fill), (Length))
```

Comment:
I think we're missing a bracket here actually

Reply:
oops, fixed